File: KDonghyeon/구현/14470 전자레인지.java
import java.io.IOException;
import java.io.BufferedReader;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int A = Integer.parseInt(br.readLine());
        int B = Integer.parseInt(br.readLine());
        int C = Integer.parseInt(br.readLine());
        int D = Integer.parseInt(br.readLine());
        int E = Integer.parseInt(br.readLine());
        int time = 0;

        if ( A <= 0 ) {
            int temp = -(A);
            time += (B*E) + (temp*C) + D;
        }
        else if ( A > 0 ) {
            B -= A;
            time = B*E;
        }
        System.out.println(time);
    }
}

File: Shinhye/CH04 Implementation/14470_전자레인지.py
a = int(input())  # 원래 고기의 온도
b = int(input())  # 목표온도
c = int(input())  # 1도당 얼어있는 고기 데우는 시간
d = int(input())  # 얼어있는 고기 해동 시간
e = int(input())  # 얼어있지 않은 고기 해동 시간

time = 0
if a < 0:
    time += ((a*-1)*c)+d+(b*e)
else:
    time += (b-a) * e

print(time)
